Update Readme (#106)
Esri/arcgis-maps-sdk-unity-samples: samples_project/Assets/SampleViewer/Samples/FeatureLayer/readme.md

![Feature layer](FeatureLayer.png)

> ## How to Setup (Feature Layer Level)

Reviewer: I'm converting it to small case as other titles in the readme are all small cases

```suggestion
## How to set up (Feature Layer Level)
```

1. Open the **FeatureLayer** Scene.
2. Click on the **ArcGISMap** Gameobject in the hierarchy window.
3. Set your API key under Authentication section in the inspector window.
4. Click play.

> ## How to Setup (Sample Viewer)

Reviewer: ```suggestion
## How to set up (Sample Viewer)
```

1. Click Play in Unity Editor.
2. Input your API key under the **API Key Drop down**.
3. Click the **Sample Drop Down** and select **Feature Layer**.

> ## How to Use

Reviewer: ```suggestion
## How to use
```

> 1. Already available is a Feature Layer of park trees, upon running the scene, these feature items will be spawned in at their default latitude and longitude and with all of the properties associated with that feature item. 

Reviewer: ```suggestion
1. Already available is a Feature Layer of park trees. Upon running the scene, these feature items will be spawned in at their default latitude and longitude, and with all of the properties associated with that feature item. 
```

> 2. Clicking on any of these objects under the **FeatureLayer** Gameobject in Unity to see it's latatude, longitude, and any properties recieved by the query.

Reviewer: ```suggestion
2. Clicking on any of these objects under the **FeatureLayer** Gameobject in Unity to see its latitude, longitude, and any properties received by the query.
```

> 3. Under the outfields drop down, by default it will be set to "Get All Outfields". Clicking on any of the other outfields will deselect "Get All Outfields" and will only get the outfield selected. Multi-select is supported. Outfields are the properties associated with the individual features. **Note** you need to reprocess the feature layer query after changes outfields to get.

Reviewer: ```suggestion
3. Under the outfields dropdown, by default it will be set to "Get All Outfields". Selecting any of the other outfields will deselect "Get All Outfields". Multi-select is supported. Outfields are the properties associated with the individual features. **Note**: after modifying the outfields, it is necessary to reprocess the feature layer query for the changes to take effect.
```

## How it works

> ### Important Note

Reviewer: ```suggestion
### Important note
```

> This sample allows users to use their own feature layer data sets to visualize using the Unity Engine. At this time, we only support `Point Layer data types`. We plan to add support for the other types in the future. We appologize for any inconvience this may cause you.

Reviewer: Correct me if I'm wrong, but I feel like we shouldn't make any promises.
```suggestion
This sample allows users to use their own feature layer data sets to visualize in Unity. At this time, we only support `Point Layer data types`.
```
